core/eolearn/tests/test_core_tasks.py

```diff
@@ -148,28 +149,69 @@ def test_io_task_pickling(filesystem, task_class):
     assert isinstance(unpickled_task, task_class)
 
 
-def test_add_rename_remove_feature(patch):
+def test_add_feature(patch: EOPatch) -> None:
     cloud_mask = np.arange(10).reshape(5, 2, 1, 1)
     feature_name = "CLOUD MASK"
-    new_feature_name = "CLM"
 
-    patch = copy.deepcopy(patch)
+    with pytest.raises(KeyError):
+        patch.mask[feature_name]
 
     patch = AddFeatureTask((FeatureType.MASK, feature_name))(patch, cloud_mask)
-    assert np.array_equal(patch.mask[feature_name], cloud_mask), "Feature was not added"
+    assert np.array_equal(patch.mask[feature_name], cloud_mask)
 
-    patch = RenameFeatureTask((FeatureType.MASK, feature_name, new_feature_name))(patch)
-    assert np.array_equal(patch.mask[new_feature_name], cloud_mask), "Feature was not renamed"
-    assert feature_name not in patch[FeatureType.MASK], "Old feature still exists"
 
-    patch = RemoveFeatureTask((FeatureType.MASK, new_feature_name))(patch)
-    assert feature_name not in patch.mask, "Feature was not removed"
+@pytest.mark.parametrize(
+    "feature, feature_data",
+    [
+        ((FeatureType.BBOX, None), BBox((24.54, 56.45, 95.4, 13.43), CRS(3857))),
+        ((FeatureType.TIMESTAMP, None), [datetime(2022, 1, 1, 10, 4, 7), datetime(2022, 1, 4, 10, 14, 5)]),
+    ],
+)
+def test_add_bbox_timestemps(feature: FeatureSpec, feature_data: Union[np.ndarray], patch: EOPatch) -> None:
+    assert patch[feature] != feature_data
+    patch = AddFeatureTask(feature)(patch, feature_data)
+    assert patch[feature] == feature_data
+
+
+def test_rename_feature(patch: EOPatch) -> None:
+    feature_name = "bands"
+    new_feature_name = "data"
```

Comment:
Additionally, I think I would prefer to see whole features here, not just names. It looks better when you call these features below, instead of hardcoding `patch.data` and such.